internal/http/proxy/proxy.go

```diff
@@ -84,3 +90,28 @@ func modifyResponse(blockFieldSuggestions *block_field_suggestions.BlockFieldSug
 		return nil
 	}
 }
+
+func logErrors(payload map[string]interface{}, log *slog.Logger) map[string]interface{} {
+	errorsPayload := payload["errors"]
+
+	if errorsPayload != nil {
```

Comment:
why not just log the raw payload? Saves needing to marshall

Reply:
also if the log format is set to json it will log as a json object anyway

Reply:
i figured i'd standardize it a bit as the log format can be different dependent on the upstream server

Reply:
also wasnt aware i could straight up do that